lib/game.rb

```diff
@@ -0,0 +1,34 @@
+class Game
+
+  attr_reader :player_2, :player_1, :player1_move, :player2_move, :rules
+
+  def initialize(player_1, player_2 = ComputerPlayer.new, rules = Rules.new)
+    @player_1 = player_1
+    @player_2 = player_2
+    @rules = rules
+  end
+
+  def self.create(player_1, player_2, rules)
+    @game = Game.new(player_1, player_2, rules)
+  end
+      
+  def self.instance
+    @game
+  end
+
+  def player_1_move(move)
```

Comment:
Maybe this method could renamed `set_player_1_move` to clearly indicate it's changing the state